ndcube/tests/test_ndcube.py

```diff
@@ -1266,7 +1266,7 @@ def test_arithmetic_add_cube_unc_nddata_unc(ndc, value):
     output_cube = ndc + value # perform the addition
     # Construct expected cube
     expected_data = ndc.data + value.data
-    expected_uncertainty = ndc.uncertainty.propagate(
+    expected_uncertainty = ndc.uncertainty.propagate(  # is this the correct way to test uncertainty? no need to test propagate
```

Comment:
This is OK in this case because, as you say, we aren't testing `propagate`, and if we are using multiple parameterizations, we might not be able to hard-code the expected result as it may be different each time.  In this case, though, we only have one test case, so strictly speaking, it's better practice to explicitly set the expected value.  But I don't think it's important to change here.